### PR TITLE
Prepare for 1.0

### PR DESCRIPTION
**Pull request by lukastaegert**
This will update all dependencies and make sure tests still run with rollup@1.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,16 +35,19 @@
...
-    "ava": "^0.12.0",
-    "coveralls": "^2.11.8",
-    "eslint": "^2.2.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "npm-run-all": "^1.5.1",
-    "nyc": "^5.6.0",
+    "ava": "^0.25.0",
+    "coveralls": "^3.0.2",
+    "eslint": "^5.10.0",
+    "eslint-config-airbnb": "^17.1.0",
```
Comment (frostney) [inline package.json]: This might be out of scope, but can't use `airbnb-base` as we don't really need the react config and `jsx-a11y`

**Review by frostney — APPROVED** [review]
👍 Thank you so much for cleaning this up.

File: package.json (hunk 1 above)
Comment (lukastaegert) [reply package.json]: I changed the ESLint config to match more or less that of Rollup

**Comment by lukastaegert** [comment]
No problem!

**Pull request merged**